### ISSUE TITLE
Kill app on background for Android 13

### ISSUE THREAD

**Issue by bernamaxim** [traccar/traccar-client-android#448]
Android 13 kill app on background, maybe notification on foreground must need runtime permission, because it not turn on automatically.

https://developer.android.com/develop/ui/views/notifications/notification-permission

![IMG_20230418_194748](https://user-images.githubusercontent.com/2969479/232782194-9860eb2e-bbf9-48ec-9eb2-4836d0e20baf.jpg)

**Comment by tananaev** [comment]
Looks like you're right. We need to enable it.

**Comment by tananaev** [comment]
Wait. I just checked the documentation again and it says explicitly:

> Note: Apps don't need to request the POST_NOTIFICATIONS permission in order to launch a foreground service. However, apps must include a notification when they start a foreground service, just as they do on previous versions of Android.

So it seems like it's not required.

**Comment by tananaev** [comment]
Just to be sure I tested it and I can see that the foreground service is running:

![Screenshot_20230418_173638](https://user-images.githubusercontent.com/1161863/232936320-89ba1463-68e0-412e-851f-22b65d9c158d.png)

**Comment by bernamaxim** [comment]
![Screenshot_2023_0419_075329](https://user-images.githubusercontent.com/2969479/232938404-d7f56939-c7b2-4d13-b4fe-931b48c645e4.jpg)

Are you sure to use Android 13 for testing?

**Comment by tananaev** [comment]
Yes, absolutely. I'm using Android 13 emulator

<img width="205" alt="Screenshot 2023-04-18 at 5 56 31 PM" src="https://user-images.githubusercontent.com/1161863/232938701-9f31d2ba-974e-466e-afa4-a7859d161a07.png">

**Comment by bernamaxim** [comment]
I have tested again to make sure.

I clear memory and cache on my device and reinstall Traccar.

About 25 minutes ago I run Traccar and sweep away (to make sure running in background)

For several updates it not problem until first 5 minutes. 

After waiting for 20 minutes, it not update anymore.

![Screenshot_20230419_081926](https://user-images.githubusercontent.com/2969479/232942533-99f08b71-d05c-4353-9a44-7691b6cc226a.jpg)

But after I opened app, it start to update again. Execute startForeground (with process: "service started" and "network online")

![Screenshot_2023_0419_082025](https://user-images.githubusercontent.com/2969479/232942656-830ad485-f64e-49c4-a492-594939255e75.jpg)

Note: 
1. I have make sure to set allow access all the time for GPS, use accurate location and turn off battery optimization.
2. I have tested for 2 device with Android OS 13.
